stubs/Unit.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 test('{name}', function () {
-    assertTrue(true);
+    $this->assertTrue(true);
```

Comment:
To be honest, I think this should use the expectation API and be:
```suggestion
    expect(true)->toBeTrue();
```